src/app/services/groupsocket/groupsocket.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { Subject } from 'rxjs/subject';
+import { Http } from '@angular/http';
+import * as io from 'socket.io-client';
+import 'rxjs/Rx';
+
+export interface UserInfo {
+  socketID : string;
+  groupUID : string;
+  username : string;
+  latitude : number;
+  longitude : number;
+
+}
+
+
+
+@Injectable()
+export class GroupSocketService {
+  //LOCALHOST
+  socketHost: string = 'http://localhost:8080';
```

Comment:
So, this `socketHost` should now be changed to the server url. You can get the with the `ConfigService`.

Reply:
Changed